draft-ietf-quic-multipath.md

```diff
@@ -885,6 +884,33 @@ two or more active paths during the connection lifetime. Different applications
 Once the implementation has decided which paths to keep alive, it can do so by sending Ping frames
 on each of these paths before the idle timeout expires.
 
+## Migration
+
+Even when the multipath extension is used, migration can happen.
+E.g. the client's source port and/or IP address may change due to changes
+in network topology or address mappings, such as caused by NAT rebinding.
+In this case the server performs path validation (see {{Section 9 of QUIC-TRANSPORT}}).
+
+Moreover, {{Section 5.1.2 of QUIC-TRANSPORT}} indicates that an endpoint
+can change the connection ID it uses for a peer to another available one
+at any time during the connection. As such a sole change of the Connection
+ID without any change in the address does indiacte a path. 
+
+If an endpoint uses a new Connection ID after an idle period
+and a NAT binding leads to a 4-tuple changes on the same packet,
+the receiving endpoint may not be able to associate the packet to
+an existing path and will therefore consider this as a new path.
+This leads to an inconsistent view of open paths at both peers,
+however, as the "old" path will not work anymore, it will be silently
+closed after the idle timeout expires. 
```

Comment:
I updated the text but I'm not really happy here. The server has to assume a new path if it can't associate a packet with new CID and new 4-tuple to an existing path but it might still try to use the old path to send data. Those packets will then be lost and it takes a timeout to detect that this is a the case. Isn't there a way for the client to say: ups I received a path validation even though I din't know my address change, so please don't send data using this old CID anymore because it's actually the same path...?